DiscordBot/Commands/EventCommands.cs

```diff
@@ -161,16 +161,14 @@ public async Task CreateEvent(CommandContext context, string eventName, string e
                 Title = "Response options"
             };
 
-
-            var i = 0;
-            foreach (var responseSet in ResponseSets)
+            for (var i = 0; i < ResponseSets.Length; i++)
             {
+                var responseSet = ResponseSets[i];
```

Comment:
Very minor but you could just inline this? You're only using it once as far as I can tell so no real advantage to having a variable 🙂